driver/image_normalize_driver.hpp

```diff
@@ -0,0 +1,347 @@
+                              const Tgpu* input,
+                              Tref* output,
+                              const Tgpu* mean,
+                              const Tgpu* stdvar)
+{
+    tensor_view_4d_t input_tv  = get_inner_expanded_4d_tv(inputDesc);
+    tensor_view_4d_t output_tv = get_inner_expanded_4d_tv(outputDesc);
+    tensor_view_4d_t mean_tv   = get_inner_expanded_4d_tv(meanDesc);
+    tensor_view_4d_t stdvar_tv = get_inner_expanded_4d_tv(stdvarDesc);
+
+    auto N         = inputDesc.GetElementSize();
+    auto C         = input_tv.size[1];
+    auto c_strides = input_tv.stride[1];
+
+    for(size_t gid = 0; gid < N; gid++)
+    {
+        auto c = gid / c_strides % C;
+
+        Tref pixel  = get4DValueAt(input, input_tv, gid);
+        Tref result = (pixel - static_cast<Tref>(mean[c + mean_tv.offset])) /
+                      static_cast<Tref>(stdvar[c + stdvar_tv.offset]);
+        set4DValueAt(output, output_tv, gid, result);
+    }
+}
+
+template <typename Tgpu, typename Tref>
+class ImageNormalizeDriver : public Driver
+{
+public:
+    ImageNormalizeDriver() : Driver()
+    {
+        miopenCreateTensorDescriptor(&inputTensorDesc);
+        miopenCreateTensorDescriptor(&outputTensorDesc);
+        miopenCreateTensorDescriptor(&meanTensorDesc);
+        miopenCreateTensorDescriptor(&stdvarTensorDesc);
+
+        data_type = miopen_type<Tgpu>{};
+    }
+
+    int AddCmdLineArgs() override;
+    int ParseCmdLineArgs(int argc, char* argv[]) override;
+    InputFlags& GetInputFlags() override { return inflags; }
+
+    int GetandSetData() override;
+    int AllocateBuffersAndCopy() override;
+
+    int RunForwardGPU() override;
+    int RunForwardCPU();
+
+    int RunBackwardGPU() override;
+    int RunBackwardCPU();
+
+    int VerifyForward() override;
+    int VerifyBackward() override;
+
+    ~ImageNormalizeDriver() override
+    {
+        miopenDestroyTensorDescriptor(inputTensorDesc);
+        miopenDestroyTensorDescriptor(outputTensorDesc);
+        miopenDestroyTensorDescriptor(meanTensorDesc);
+        miopenDestroyTensorDescriptor(stdvarTensorDesc);
+    }
+
+private:
+    InputFlags inflags;
+    int forw;
+
+    miopenTensorDescriptor_t inputTensorDesc;
+    miopenTensorDescriptor_t outputTensorDesc;
+    miopenTensorDescriptor_t meanTensorDesc;
+    miopenTensorDescriptor_t stdvarTensorDesc;
+
+    std::unique_ptr<GPUMem> input_gpu;
+    std::unique_ptr<GPUMem> output_gpu;
+    std::unique_ptr<GPUMem> mean_gpu;
+    std::unique_ptr<GPUMem> stdvar_gpu;
+
+    std::vector<Tgpu> input_host;
+    std::vector<Tgpu> output_host;
+    std::vector<Tgpu> mean_host;
+    std::vector<Tgpu> stdvar_host;
+
+    std::vector<Tref> out_ref;
+};
+
+template <typename Tgpu, typename Tref>
+int ImageNormalizeDriver<Tgpu, Tref>::AddCmdLineArgs()
+{
+    inflags.AddInputFlag("forw", 'F', "1", "Run only the forward pass", "int");
+    inflags.AddTensorFlag("input", 'I', "1x3x96x96", "Input Tensor Size");
```

Comment:
Please add example to description

Reply:
Same as other drivers. This will be added by the framework itself